source/nsdynmemLIB/nsdynmemLIB.c

```diff
@@ -290,178 +238,49 @@ void *ns_dyn_mem_alloc(int16_t alloc_size)
 #endif
 }
 
+void *ns_dyn_mem_alloc(int16_t alloc_size)
+{
+    return ns_dyn_mem_internal_alloc(alloc_size, -1);
```

Comment:
Maybe worth a comment, as this is the top-level function people will glance at. "Allocates from end of heap".